### PR TITLE
Zủbeo jí sa rải da. nhaq!

### PR DESCRIPTION
**Pull request by lunarythia**

### CODE REVIEW

File: Hủaıbaı/assets/minecraft/lang/toa_tqg.json [hunk 1]
```diff
@@ -118,8 +118,32 @@
...
+  "options.difficulty.hardcore": "Pỏ Hỉaodaı ga Sỏqne",
+  "options.fov": "Kảqjıeqgıu",
+  "options.fov.min": "Nẻfaı",
+  "options.fov.max": "Kảqsoqne",
+  "soundCategory.master": "Jỏqlaqcaı",
+  "soundCategory.music": "Gỉaq",
+  "soundCategory.record": "Gỉaqkea ru Lảqkea",
+  "soundCategory.weather": "Sẻoqtue",
+  "soundCategory.hostile": "Pảıgıqruo Mỉerıoq",
+  "soundCategory.neutral": "Pảıruo Mỉerıoq",
```
Comment (robintown) [inline Hủaıbaı/assets/minecraft/lang/toa_tqg.json]: I think these would be worth simplifying to just 'Ỉoma' and 'Nỉaı'

File: Hủaıbaı/assets/minecraft/lang/toa_tqg.json [hunk 2]
```diff
@@ -118,8 +118,32 @@
...
+  "soundCategory.player": "Lủaqche",
```
Comment (robintown) [inline Hủaıbaı/assets/minecraft/lang/toa_tqg.json]: The rest of the translation uses 'pỏq' for 'player'

File: Hủaıbaı/assets/minecraft/lang/toa_tqg.json [hunk 3]
```diff
@@ -118,8 +118,32 @@
...
+  "options.difficulty.online": "Pỏ Lủeqce ga Nỏacaı",
+  "options.difficulty.peaceful": "Sỏıcıaruo",
+  "options.difficulty.easy": "Fủı",
+  "options.difficulty.normal": "Nẻfaı",
+  "options.difficulty.hard": "Nỏa",
+  "options.difficulty.hardcore": "Pỏ Hỉaodaı ga Sỏqne",
+  "options.fov": "Kảqjıeqgıu",
+  "options.fov.min": "Nẻfaı",
+  "options.fov.max": "Kảqsoqne",
+  "soundCategory.master": "Jỏqlaqcaı",
```
Comment (robintown) [inline Hủaıbaı/assets/minecraft/lang/toa_tqg.json]: We could avoid creating a special word for this purpose with a serial verb like 'Jóq nẻ lảqcaı'

File: Hủaıbaı/assets/minecraft/lang/toa_tqg.json [hunk 4]
```diff
@@ -118,8 +118,32 @@
   "disconnect.genericReason": "%s",
   "disconnect.quitting": "Chủfaq shảı lủaq",
   "options.language": "Zủ...",
+  "options.sounds": "Gỉaq ru Lảq...",
```
Comment (robintown) [inline Hủaıbaı/assets/minecraft/lang/toa_tqg.json]: So unfortunately inserting 'ru' here doesn't work like you're expecting, and instead this means "things that are both music and sound". We could do something like 'Baq gỉaq ru/roı baq lảq', but in the spirit of keeping all the button labels succinct and free of noun phrases, I suggest we sidestep the problem by just using 'Lảq…' or 'Hủo…' as the label

File: Hủaıbaı/assets/minecraft/lang/toa_tqg.json [hunk 5]
```diff
@@ -118,8 +118,32 @@
...
+  "options.sounds.title": "Pỏ Gỉaq ru Lảq ga Kẻajuao",
```
Comment (robintown) [inline Hủaıbaı/assets/minecraft/lang/toa_tqg.json]: a few notes:
1. Pỏ takes a noun phrase as its complement, so this would rather be something like 'Pỏ baq lảq ga kẻajuao'
2. Please keep words uncapitalized (other than the first word in a string), for consistency with the rest of the translation
3. I'm not super fond of the word kẻajuao, I've added cảqjuao to the dictionary just now for this purpose

File: Hủaıbaı/assets/minecraft/lang/toa_tqg.json [hunk 6]
```diff
@@ -118,8 +118,32 @@
...
   "record.nowPlaying": "Nỉ gỉaq ku: %s",
   "options.languageWarning": "Dảı bû mủo dủe sé tủa nỏ zủ zay",
+  "options.online": "Zỉqrıaq...",
+  "options.online.title": "Pỏ Zỉqrıaq ga Kẻajuao",
+  "options.difficulty": "Nỏacaı",
```
Comment (robintown) [inline Hủaıbaı/assets/minecraft/lang/toa_tqg.json]: 'Cảı' means 'strong' rather than 'strength', so I'd suggest 'Nẻ nỏa' for this purpose

File: Hủaıbaı/assets/minecraft/lang/toa_tqg.json [hunk 7]
```diff
@@ -118,8 +118,32 @@
   "disconnect.genericReason": "%s",
   "disconnect.quitting": "Chủfaq shảı lủaq",
   "options.language": "Zủ...",
+  "options.sounds": "Gỉaq ru Lảq...",
+  "options.sounds.title": "Pỏ Gỉaq ru Lảq ga Kẻajuao",
   "record.nowPlaying": "Nỉ gỉaq ku: %s",
   "options.languageWarning": "Dảı bû mủo dủe sé tủa nỏ zủ zay",
+  "options.online": "Zỉqrıaq...",
+  "options.online.title": "Pỏ Zỉqrıaq ga Kẻajuao",
```
Comment (robintown) [inline Hủaıbaı/assets/minecraft/lang/toa_tqg.json]: Same comments as on 'Pỏ Gỉaq ru Lảq ga Kẻajuao'

File: Hủaıbaı/assets/minecraft/lang/toa_tqg.json [hunk 8]
```diff
@@ -118,8 +118,32 @@
...
+  "options.difficulty": "Nỏacaı",
+  "options.difficulty.online": "Pỏ Lủeqce ga Nỏacaı",
```
Comment (robintown) [inline Hủaıbaı/assets/minecraft/lang/toa_tqg.json]:
```suggestion
  "options.difficulty.online": "Pỏ lúeqce ga nẻ nỏa",
```

File: Hủaıbaı/assets/minecraft/lang/toa_tqg.json [hunk 9]
```diff
@@ -118,8 +118,32 @@
...
+  "options.difficulty.peaceful": "Sỏıcıaruo",
```
Comment (robintown) [inline Hủaıbaı/assets/minecraft/lang/toa_tqg.json]: I think just 'Sỏıcıa' would get the point across here as well

File: Hủaıbaı/assets/minecraft/lang/toa_tqg.json [hunk 10]
```diff
@@ -118,8 +118,32 @@
...
+  "options.difficulty.easy": "Fủı",
+  "options.difficulty.normal": "Nẻfaı",
+  "options.difficulty.hard": "Nỏa",
+  "options.difficulty.hardcore": "Pỏ Hỉaodaı ga Sỏqne",
+  "options.fov": "Kảqjıeqgıu",
+  "options.fov.min": "Nẻfaı",
+  "options.fov.max": "Kảqsoqne",
+  "soundCategory.master": "Jỏqlaqcaı",
+  "soundCategory.music": "Gỉaq",
+  "soundCategory.record": "Gỉaqkea ru Lảqkea",
```
Comment (robintown) [inline Hủaıbaı/assets/minecraft/lang/toa_tqg.json]: Similarly to 'Gỉaq ru Lảq', we could shorten this to just 'Lảqkea' or something

**Review by robintown — COMMENTED** [review]
Yay, I was dreading the day when I'd translate the options screen, so I'm glad someone else has started it for me :)

File: Hủaıbaı/assets/minecraft/lang/toa_tqg.json (hunk 5 above)
Comment (lunarythia) [reply Hủaıbaı/assets/minecraft/lang/toa_tqg.json]: I think I capitalised it because the original had something along the lines of "Music and Sound Options" iirc

**Comment by lunarythia** [comment]
nãi bi, mỏi jí dôegitua jí tú rải da

**Review by robintown — APPROVED** [review]
:toaqhua:
